src/engine/controls/cuecontrol.cpp

```diff
@@ -50,6 +51,9 @@ CueControl::CueControl(QString group,
     m_pPrevBeat = ControlObject::getControl(ConfigKey(group, "beat_prev"));
     m_pNextBeat = ControlObject::getControl(ConfigKey(group, "beat_next"));
     m_pClosestBeat = ControlObject::getControl(ConfigKey(group, "beat_closest"));
+    m_pLoopStartPosition = new ControlProxy(group, "loop_start_position");
```

Comment:
if you add "this" as third argument the object is deleted with the CueControl Object.